addons/web/static/src/js/views/form_view.js

```diff
@@ -747,8 +747,10 @@ var FormView = View.extend(common.FieldManagerMixin, {
                 self.to_view_mode();
                 core.bus.trigger('do_reload_needaction');
                 core.bus.trigger('form_view_saved', self);
+            }).always(function() {
+                self.enable_button();
             });
-        }).always(function(){
+        }).fail(function(){
```

Comment:
I don't understand why the previous code did not work.  But other than that, this commit looks fine